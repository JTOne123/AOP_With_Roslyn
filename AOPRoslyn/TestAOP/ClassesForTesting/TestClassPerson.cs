using System;
using System.Collections.Generic;
using System.Text;

namespace TestAOP
{
    class TestClassPerson
    {
        public TestClassPerson(string firstName, string lastName)
        {
            FirstName = firstName;
            LastName = lastName;
        }

        public string FirstName { get; }
        public string LastName { get; }

        public string Name()
        {
            return FirstName + " " + LastName;
        }
        
        public int Length()
        {
            return Name().Length;
        }
    }
}
