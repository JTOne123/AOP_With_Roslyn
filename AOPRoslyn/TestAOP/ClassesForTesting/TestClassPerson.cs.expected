using System;
using System.Collections.Generic;
using System.Text;

namespace TestAOP
{
    class TestClassPerson
    {
        public TestClassPerson(string firstName, string lastName)
        {
            FirstName = firstName;
            LastName = lastName;
        }

        public string FirstName { get; }
        public string LastName { get; }

        public string Name()
{Console.WriteLine("start TestClassPerson_Name_17");
#line 20
return FirstName + " " + LastName;
Console.WriteLine("end TestClassPerson_Name_17");}        
        public int Length()
{Console.WriteLine("start TestClassPerson_Length_22");
#line 25
return Name().Length;
Console.WriteLine("end TestClassPerson_Length_22");}    }
}
